input/docs/getting-started/index.md

```diff
@@ -17,7 +17,7 @@ Let's create a simple application demonstrating a number of ReactiveUI functiona
 We will create a WPF application, which will allow us to search through Flickr public images.   
 The full code of the application is shown at the end of this chapter, and we will show relevant snippets as we go.
 
-In Visual Studio create a new WPF application (.Net 4.5 or above)
+In Visual Studio, create a new WPF application (.NET 4.5 or above).
```

Comment:
I believe we increased that to like 4.6.1 now.

Reply:
Looks good otherwise.

Reply:
https://github.com/reactiveui/ReactiveUI/blob/f2290c290f189ab349dcdae0a5cc752a69fb06f7/src/ReactiveUI.Wpf/ReactiveUI.Wpf.csproj#L9

Seems like it!